waywiser_nav2/launch/nav2_bringup_all.launch.py

```diff
@@ -0,0 +1,341 @@
+# Copyright (c) 2018 Intel Corporation
```

Comment:
Does this actually come from intel or was this pasted by accident?

Reply:
It's coming from the nav2 bringup package source file: https://github.com/ros-planning/navigation2/blob/1ff1303537dfaceaa953faff7530febace585f83/nav2_bringup/launch/bringup_launch.py#L1